### ISSUE TITLE
Arrow ligatures using box drawing characters

### ISSUE THREAD

**Issue by schuelermine** [microsoft/cascadia-code#302]
# Description of the new feature/enhancement

Add  the ligature `─>` (codepoint sequence U+2500, U+003E).
It should connect with `─` (U+2500)

# Also

It's worth thinking about whether to add this kind of ligature to other box drawing characters.

**Comment by aaronbell** [comment]
Hi! So the glyphs that you're trying to connect up to the box drawing characters are actually coding ligatures. They're intended to align with the mathematical symbols that they are replacing. So they're not really meant to align with the box drawing characters. 

As far as I've been able to tell, the various arrow symbols, such as `→` (uni2192) would be a more likely candidate for alignment with the box drawing characters, but I have not observed alignment with any of those either. Can you point me to an example of another font that aligns any arrows with the box drawing characters?

**Comment by schuelermine** [comment]
No. The ligature I'm proposing *contains* a box drawing character.

**Comment by schuelermine** [comment]
So `─->` would be disconnected, and `──>` would be connected.

**Comment by aaronbell** [comment]
Ah, I see! Thanks :)

**Comment by aaronbell** [comment]
Hi. Question for you. I'm currently implementing the arrows system like Fira Code has (see #276) and am thinking more about your request. Would that be sufficient for your needs, or are you attached to having them connect with the boxdrawing characters?

And as a follow up, the arrow system includes > < | / as different options. Are you considering all those as well?

**Comment by schuelermine** [comment]
I guess it's not really appropriate to call them needs, it's not like I desperately require this feature. But I don't think a Fira-style arrow system is a replacement for my idea.

About all the arrow system components, I don't think including all of these would make sense, necessarily. I think my idea is more about a distinct feature where `->` and maybe `-|` (box drawing characters replaced with dash since I'm on mobile) get replaced.

But of course, you could just integrate the box drawing characters into the arrow system. In that case, I'd suggest that it only works when the other stem parts of the arrow are also box drawing characters. I don't think mixing them with dashes is a good idea.
